scheduler/src/cook/mesos/api.clj

```diff
@@ -687,7 +687,7 @@
                     base {:task_id (:instance/task-id instance)
                           :hostname hostname
                           :ports (:instance/ports instance)
-                          :backfilled (:instance/backfilled? instance false)
+                          :backfilled false ;; Backfill has been deprecated
```

Comment:
I'm hesitant to do that because their may be clients that expect it. The joys(?) of backwards compatibility.